def main
  break             # Error: break outside loop
  return: 42
